### PR TITLE
docs:add supernode gc doc

### PR DESCRIPTION
**Pull request by yunfeiyanggzq**
Signed-off-by: yunfeiyangbuaa <yunfeiyang@buaa.edu.cn>

### CODE REVIEW

**Comment by Starnop** [comment]
@yunfeiyanggzq IMOP, maybe it's better to add a supernode config doc under the dir `docs/config` just as #802 and make a link to it. WDYT?

In addition, #802 has not been updated for a long time. Are you willing to open a new PR  to add config docs for `dfademon`, `dfget` and `supernode`?

**Comment by yunfeiyanggzq** [comment]
> 
> 
> @yunfeiyanggzq IMOP, maybe it's better to add a supernode config doc under the dir `docs/config` just as #802 and make a link to it. WDYT?
> 
> In addition, #802 has not been updated for a long time. Are you willing to open a new PR to add config docs for `dfademon`, `dfget` and `supernode`?

ok, I am busy recently,my work may be slow but I will do as soon as possible

File: FAQ.md [hunk 1]
```diff
@@ -313,4 +313,21 @@ If you are in China,docker container uses UTC time(Coordinated Universal Time) a
 
 ## How to join Dragonfly as a member
 
-Please check the [CONTRIBUTING.md](CONTRIBUTING.md#join-dragonfly-as-a-member)
\ No newline at end of file
+Please check the [CONTRIBUTING.md](CONTRIBUTING.md#join-dragonfly-as-a-member)
+
+## Can I config the gc parameter
```
Comment (allencloud) [inline FAQ.md]: s/config/configure

File: FAQ.md [hunk 2]
```diff
@@ -313,4 +313,21 @@ If you are in China,docker container uses UTC time(Coordinated Universal Time) a
...
+
+Yes, we can config the supernode gc parameter.
```
Comment (allencloud) [inline FAQ.md]: s/config/configure

File: FAQ.md [hunk 3]
```diff
@@ -313,4 +313,21 @@ If you are in China,docker container uses UTC time(Coordinated Universal Time) a
...
+
+There are some parameters we can config in supernode.
```
Comment (allencloud) [inline FAQ.md]: s/config/configure

File: FAQ.md [hunk 4]
```diff
@@ -313,4 +313,21 @@ If you are in China,docker container uses UTC time(Coordinated Universal Time) a
...
+Name                           | Default Value | Description
+------------------------------ | ------------- | ----------
+gc-initial-delay               | 6s            | it is the delay time from the start to the first GC execution.
+gc-meta-interval               | 2min          | it is the interval time to execute the GC meta.
+task-expire-time               | 3min          | when a task is not accessed within the time,it will be treated to be expired.
+peer-gc-delay                  | 3min          | it is the delay time to execute the GC after the peer has reported the offline.
+
+In supernode ,gc will begin `gc-initial-delay` time after supernode works.Then supernode will run peer-gc goroutine and task-gc goroutine every  `gc-meta-interval` time.
+If a task isn't accessed by dfgets in `task-expire-time` time, task-gc goroutine will gc this task, but now supernode only gc the task information in memory.
+If a peer reports it servers down, peer-gc goroutine will gc this peer after `peer-gc-delay` time.
```
Comment (allencloud) [inline FAQ.md]: `If a peer reports it servers down` should be more clear. Actually I cannot quite follow it.

Comment (yunfeiyanggzq) [reply FAQ.md]: I agree with you,I will fix it

File: FAQ.md (hunk 1 above)
Comment (yunfeiyanggzq) [reply FAQ.md]: done

File: FAQ.md (hunk 2 above)
Comment (yunfeiyanggzq) [reply FAQ.md]: done

File: FAQ.md (hunk 3 above)
Comment (yunfeiyanggzq) [reply FAQ.md]: done

File: FAQ.md (hunk 4 above)
Comment (yunfeiyanggzq) [reply FAQ.md]: done

**Comment by yunfeiyanggzq** [comment]
@Starnop please review this pr

**Pull request closed without merging**